src/components/Sidebar.tsx

```diff
@@ -27,6 +28,17 @@ export const Sidebar = (): JSX.Element => {
   return (
     <nav role="navigation">
       <List>
+        {isFlagshipApp() ||
+          (flag('settings.flagship-mode') && (
+            <MenuList title={t('Nav.header_flagship')}>
+              <MenuItemNavLink
+                to={routes.lockScreen}
+                primary={t('Nav.primary_lock_screen')}
+                icon={PeopleIcon} // @todo icon
```

Comment:
is the PR ready for review?

Reply:
yes I just need an icon to be merged in cozy-ui but everything is review ok

Reply:
@trollepierre https://github.com/cozy/cozy-ui/pull/2214